test/service/test_manifest.py

```diff
@@ -428,11 +428,11 @@ def test_manifest_zarr(self):
                     lines = response.content.decode().splitlines()
                     tsv_file = csv.DictReader(lines, delimiter='\t')
                     rows = list(tsv_file)
-                    # self.assertEqual(len(rows), 13)  # 12 related file, one original
-                    self.assertEqual({f['uuid'] for f in expected}, {row['file_uuid'] for row in rows})
-                    # 2 pairs of files out of the 13 have matching checksums
-                    self.assertEqual({f['sha256'] for f in expected}, {row['file_sha256'] for row in rows})
-                    self.assertEqual({f['crc32c'] for f in expected}, {row['file_crc32c'] for row in rows})
+                    files = [dict(crc32c=row['file_crc32c'],
```

Comment:
I think you should augment `expected` to contain all of the fields that are in rows. Then you don't have to do this comprehension, and also the assert is stronger.

Reply:
Good call. However, I think that augmenting `expected` to include all the manifest `csv` is beyond the scope of this test. I discussed your suggested changes with Hannes, to which he pointed out there are already tests that assert the `csv` response verbatim. Hannes did recommend adding an additional field (`specimen_from_organism.organ`) not obtained from the `files` entity, given that we are testing for inclusion of `related_files` in the manifests.